docs/source/faq.rst

```diff
@@ -714,3 +714,66 @@ Optuna may sometimes suggest parameters evaluated in the past and if you would l
 
     study = optuna.create_study()
     study.optimize(objective, n_trials=100)
+
+How can I sample the best expected parameters based on the evaluated trials?
+----------------------------------------------------------------------------
+
+As the optimization often overtunes the parameters of an objective function with an observational noise, it is convenient to sample the best parameters based on the debiased objective function.
+:class:`~optuna.samplers.GPSampler` enables us to sample the best expected parameters using the mean estimation of parameters.
+Note that the best expected parameters are defined as :math:`\mathrm{arg}\min_{x \in X} \mathbb{E}_{f}[f(x)]` where :math:`f` follows the Gaussian process.
+However, as we made the interface of :class:`~optuna.samplers.GPSampler` as simple as possible, we need to tweak it as follows:
+
+.. code-block:: python
+
+    import numpy as np
+
+    import optuna
+    import optuna._gp.acqf as acqf
+    from optuna._gp.optim_mixed import optimize_acqf_mixed
+
+
+    class BestExpectedParamSampler(optuna.samplers.GPSampler):
+        # Inherit `GPSampler` so as to use another acquisition function.
+
+        def _optimize_acqf(self, acqf_params, best_params):
+            # Copy the parameters necessary for acquisition functions.
+            acqf_params_kwargs = acqf_params.__dict__.copy()
+            # Use `UCB` with beta=0 to estimate the mean of an input.
+            acqf_params_kwargs.update(acqf_type=acqf.AcquisitionFunctionType.UCB, beta=0.0)
+            # Create another AcquisitionFunctionParams for mean estimations.
+            acqf_params = acqf.AcquisitionFunctionParams(**acqf_params_kwargs)
+            # Optimize the mean estimator. (Use self._rng.rng optionally for reproducibility.)
+            best_normalized_params, _ = optimize_acqf_mixed(acqf_params)
+            return best_normalized_params
+
+
+    def objective(trial, evaluate_func=True):
+        X = np.array([trial.suggest_float(f"x{i}", -5, 5) for i in range(2)])
+        if evaluate_func:
+            # Could be a very expensive evaluation in practice such as a training of DNN.
```

Comment:
I agree that DNN is a widely used abbreviation, but `deep neural networks` sounds more reader-friendly.
```suggestion
            # Could be a very expensive evaluation in practice such as a training of deep neural networks.
```